config/app.default.php

```diff
@@ -370,5 +370,21 @@
      */
     'Accept' => [
         'html' => filter_var(env('ACCEPT_HTML', 'false'), FILTER_VALIDATE_BOOLEAN),
+    ],
+
+    /**
+     * Additional plugins to load with this format: 'PluginName' => load options array
+     * Where options array may contain
+     *
+     * - `debugOnly` - boolean - (default: false) Whether or not you want to load the plugin when in 'debug' mode only
+     * - `bootstrap` - boolean - (default: false) Whether or not you want the $plugin/config/bootstrap.php file loaded.
+     * - `routes` - boolean - (default: false) Whether or not you want to load the $plugin/config/routes.php file.
+     * - `ignoreMissing` - boolean - (default: false) Set to true to ignore missing bootstrap/routes files.
+     * - `autoload` - boolean - (default: false) Whether or not you want an autoloader registered
+     */
+    'Plugins' => [
+        'DebugKit' => ['debugOnly' => true, 'bootstrap' => true],
+        'Migrations' => ['debugOnly' => true],
```

Comment:
**Migrations** should be loaded even if debug mode is turned off. It is required in order to run `bin/cake migrations migrate` on production systems when some update to the database schema is available.

Instead, it should probably be loaded only in CLI environments. Looking at [CakePHP skeleton app](https://github.com/cakephp/app/blob/master/config/bootstrap_cli.php), both **Migrations** and **Bake** are loaded in `config/bootstrap_cli.php`.